pom.xml

```diff
@@ -88,7 +88,7 @@
         <jackson1.version>1.9.13</jackson1.version>
         <jackson2.version>2.12.1</jackson2.version>
         <lucene-solr.version>8.6.2</lucene-solr.version>
-        <elasticsearch.version>7.10.1</elasticsearch.version>
+        <elasticsearch.version>7.10.2</elasticsearch.version>
```

Comment:
This commit only bumps elasticsearch-rest-client version but not the elasticsearch server version used in tests.

Reading https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/java-rest-high-compatibility.html further, seems generally it is more recommended to use an old version of the client against a newer version of the server, rather than the converse. Thus I am thinking if we should use the oldest client version that we support.